lab_2_keywords_cooccurrence/main.py

```diff
@@ -17,7 +28,16 @@ def extract_phrases(text: str) -> Optional[Sequence[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not check_type_and_not_empty(text, str):
+        return None
+    pattern = re.compile(r'''
```

Comment:
thanks, but I will ask it on face-to-face

Reply:
Ok. No problem